src/ldp/http/SimpleSparqlUpdateBodyParser.ts

```diff
@@ -23,17 +24,21 @@ export class SimpleSparqlUpdateBodyParser extends BodyParser {
 
   public async handle(input: HttpRequest): Promise<SparqlUpdatePatch> {
     try {
-      const sparql = await readableToString(input);
+      // Note that readableObjectMode is only defined starting from Node 12
```

Comment:
I would definitely be ok with that since that would make some things cleaner. :D But I'll leave it up to @RubenVerborgh what we actually want to support.

Reply:
Actually, let's use https://www.npmjs.com/package/readable-stream for a consistent and semver'ed `Stream` implementation across all Node versions.

Reply:
Going to change that in a separate PR then, because this will change several classes